release-please-config.json

```diff
@@ -25,6 +25,17 @@
 			"include-v-in-tag": true,
 			"include-component-in-tag": true,
 			"extra-files": ["version.go"]
+		},
+		"sdk/@launchdarkly/observability-dotnet": {
+			"release-as": "0.1.0",
```

Comment:
To be removed after this release is published, and subsequently the number will increment as expected.

Reply:
why is this necessary? is it because otherwise the default is to release a 1.0.0?

Reply:
Yes. It would try to release 1.0.0.